tests/test_client.py

```diff
@@ -26,46 +25,32 @@ def test_init(api_client: ApiClient):
     assert api_client
 
 
-def test_timeout_default(httpx_mock: HTTPXMock):
-    httpx_mock.add_response(
-        match_extensions={
-            "timeout": {
-                "connect": DEFAULT_TIMEOUT_CONFIG.connect,
-                "read": DEFAULT_TIMEOUT_CONFIG.read,
-                "write": DEFAULT_TIMEOUT_CONFIG.write,
-                "pool": DEFAULT_TIMEOUT_CONFIG.pool,
-            }
-        },
-        status_code=httpx.codes.BAD_GATEWAY,
+def test_base_url_override(respx_mock: respx.mock):
+    respx_mock.get(re.compile(r"^https://test/repos/foo/bar/import/issues")).mock(
+        return_value=httpx.Response(
+            httpx.codes.OK, text=get_fixture("response-multiple-check-status-of-multiple-issues.json")
```

Comment:
Why do you use `httpx.codes` instead of `HTTPStatus` from Python std library?

Reply:
No real good reason. The author does it this way in the `RESPX` documentation, and I thought it was a good thing for consistency if I'm using `httpx` at all.

Reply:
You could also use `HTTPMethod` in some tests instead of hardcoded string.